src/bin/css3-parser.rs

```diff
@@ -0,0 +1,160 @@
+use anyhow::{anyhow, Result};
+use gosub_engine::byte_stream::{ByteStream, Encoding, Stream};
+use gosub_engine::css3;
+use gosub_engine::css3::location::Location;
+use gosub_engine::css3::parser_config::ParserConfig;
+use gosub_engine::css3::{Css3, Error};
+use simple_logger::SimpleLogger;
+use std::fs;
+use std::process::exit;
+
+fn bail(message: &str) -> ! {
+    println!("{message}");
+    exit(1);
+}
+
+fn main() -> Result<()> {
```

Comment:
This binary allows you to simply parse, and/or tokenize css tree's found online or on file.